libs/components/lists/src/lib/modules/repeater/repeater.component.ts

```diff
@@ -333,4 +349,40 @@ export class SkyRepeaterComponent
       return item.tag !== undefined;
     });
   }
+
+  private updateRole() {
+    // Default to list role.
+    let autoRole: SkyRepeaterRoleType = 'list';
+    const roleMap: Record<SkyRepeaterRoleType, SkyRepeaterItemRolesType> = {
+      list: { item: 'listitem', title: undefined, content: undefined },
+      listbox: { item: 'option', title: undefined, content: undefined },
+      grid: { item: 'row', title: 'rowheader', content: 'gridcell' },
+    };
+    const hasSelectables = this.items?.some((item) => item.selectable);
+    const interactionSelector = [
+      'a[href]',
+      'button',
+      'select',
+      'textarea',
+      'input',
+    ]
+      .map((selector) => `sky-repeater-item-right ${selector}`)
```

Comment:
Is `sky-repeater-item-right` a component element? I'm not seeing it anywhere. Should this be `.sky-repeater-item-right`?

Reply:
Fixed.